dev-aws/kafka-shared-msk/payment-platform/payment.tf

```diff
@@ -76,10 +76,30 @@ resource "kafka_topic" "payment_v1_public_events_cbc_topup_v3" {
   }
 }
 
+//All public events will go here. It's meant to be used by ops downstream.
+//It should not be consume by a particular domain team.
+resource "kafka_topic" "payment_v1_public_events" {
+  name               = "payment-platform.payment.v1.public.events"
+  replication_factor = 3
+  partitions         = 5
```

Comment:
Just checking: is this topic migrated from an existing cluster? If true, it should have exactly the same number of partitions.

Reply:
nope. it's a new topic